FirebaseInstallations/Source/Library/InstallationsIDController/FIRInstallationsIDController.m

```diff
@@ -44,6 +44,8 @@
 
 NSTimeInterval const kFIRInstallationsTokenExpirationThreshold = 60 * 60;  // 1 hour.
 
+static NSString *const kStorageKeychainService = @"com.firebase.FIRInstallations.installations";
```

Comment:
Could this be `kKeychainService`? When I see "Storage", my initial thought is it has something to do with FirebaseStorage.

Reply:
Agreed, "Storage" is a bit extra here. Will update.